src/templates/rule.js

```diff
@@ -87,6 +87,24 @@ const Rule = ({ data, location }) => {
     }
   };
 
+  const getRelatedRule = (relatedRuleUri) => {
+    var relatedRule = data.relatedRules.nodes.find(
+      (r) => r.frontmatter.uri === relatedRuleUri
+    );
+    if (!relatedRule) {
+      data.relatedRulesFromRedirects.nodes.forEach((r) => {
```

Comment:
let's use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of instead of foreach and then return immediately

Reply:
Done - see latest commit